latiss/transmission_optics/19700101T000000.ecsv

```diff
@@ -0,0 +1,403 @@
+# %ECSV 1.0
+# ---
+# datatype:
+# - {name: wavelength, unit: nm, datatype: float64}
+# - {name: efficiency, unit: '%', datatype: float64}
+# meta: !!omap
+# - {MODE: DETECTOR}
+# - {TYPE: transmission_optics}
+# - {CALIBDATE: '1970-01-01T00:00:00'}
+# - {INSTRUME: LATISS}
+# - {OBSTYPE: transmission_optics}
+# - {CALIBCLS: lsst.ip.isr.IntermediateOpticsTransmissionCurve}
```

Comment:
Pulling from the discussion below: this file also should have a keyword noting the data source.